sklearn/neighbors/tests/test_kde.py

```diff
@@ -247,3 +252,61 @@ def test_check_is_fitted(method):
 
     with pytest.raises(NotFittedError):
         getattr(kde, method)(X)
+
+
+@pytest.mark.parametrize("bandwidth", ["scott", "silvermann", 0.1])
+def test_bandwidth(bandwidth):
+    n_samples, n_features = (100, 3)
+    rng = np.random.RandomState(0)
+    X = rng.randn(n_samples, n_features)
+    kde = KernelDensity(bandwidth=bandwidth).fit(X)
+    samp = kde.sample(100)
+    kde_sc = kde.score_samples(X)
+    assert X.shape == samp.shape
+    assert kde_sc.shape == (n_samples,)
+
+    # Test that the attribute self.bandwidth_ has the expected value
+    if bandwidth == "scott":
+        h = X.shape[0] ** (-1 / (X.shape[1] + 4))
+    elif bandwidth == "silvermann":
+        h = (X.shape[0] * (X.shape[1] + 2) / 4) ** (-1 / (X.shape[1] + 4))
+    else:
+        h = bandwidth
+    assert kde.bandwidth_ == pytest.approx(h)
+
+
+@pytest.mark.parametrize(
+    "bandwidth, exc_type, exc_msg",
+    [
+        (
+            "str test",
+            ValueError,
+            r"When `bandwidth` is a string, it should be one of: "
+            r"scott,silvermann. Got str test instead.",
```

Comment:
I commented when you pushed. I would a add a space after the comma and show the repr as suggested above.